assets/js/screens/Work/List.jsx

```diff
@@ -1,53 +1,101 @@
-import React from "react";
+import React, { useState } from "react";
 import Layout from "../Layout";
-import SearchResultItem from "../../components/Search/ResultItem";
 import { ReactiveList } from "@appbaseio/reactivesearch";
-import { IIIFProvider } from "../../components/IIIF/IIIFProvider";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
+import WorkCardItem from "../../components/Work/UIWorkCardItem";
+import WorkListItem from "../../components/Work/UIWorkListItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const ScreensWorkList = () => {
+  const [isListView, setIsListView] = useState(false);
+
+  const getWorkItem = (res) => {
+    return {
+      id: res._id,
+      title: res.title,
+      updatedAt: res.modified_date,
+      representativeImage: res.representative_file_set,
+      manifestUrl: res.iiif_manifest,
+      published: res.published,
+      visibility: res.visibility_term,
+      descriptiveMetadata: { title: res.title, description: res.description },
+      fileSets: res.file_sets,
+      accessionNumber: res.accession_number,
+      workType: res.work_type,
+    };
```

Comment:
Let's make this match the code mentioned above.  We want to send in as few, lightweight props, as necessary to maintain performance.